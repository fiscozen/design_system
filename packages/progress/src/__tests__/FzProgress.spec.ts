import { describe, it, vi } from 'vitest'
import { mount } from '@vue/test-utils'
import { FzProgress } from '..'

describe.concurrent('FzProgress', () => {
  it('matches snaphost', async ({ expect }) => {
    const wrapper = mount(FzProgress, {
      props: {},
      slots: {}
    })

    expect(wrapper.html()).toMatchSnapshot()
  })
})
