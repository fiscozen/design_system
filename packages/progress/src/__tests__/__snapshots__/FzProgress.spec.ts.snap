// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`FzProgress > matches snaphost 1`] = `
"<div class="flex items-center justify-center w-[25px] h-[25px]" style="--fa-animation-duration: 0.86s; --fa-animation-timing: cubic-bezier(0.4, 0.15, 0.6, 0.85);"><svg class="svg-inline--fa fa-spinner-third fa-spin fa-lg h-[20px]" aria-hidden="true" focusable="false" data-prefix="fas" data-icon="spinner-third" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512">
    <path class="" fill="currentColor" d="M224 32c0-17.7 14.3-32 32-32C397.4 0 512 114.6 512 256c0 46.6-12.5 90.4-34.3 128c-8.8 15.3-28.4 20.5-43.7 11.7s-20.5-28.4-11.7-43.7c16.3-28.2 25.7-61 25.7-96c0-106-86-192-192-192c-17.7 0-32-14.3-32-32z"></path>
  </svg></div>"
`;
